### ISSUE TITLE
Feature request: Option to change the output name

### ISSUE THREAD

**Issue by RickR2H** [glenn2223/vscode-live-sass-compiler#107]
Hi,

Thank for this great extension and I use it a lot! I've a small question... In certain situations it would be really helpful if we could change the output name. I know this is nearly impossible if there are multiple entry points. So I have a proposition. Why not make an option to remove a prefix when the file is generated? Example below: 

{
    "liveSassCompile.settings.formats":[
        {
            "format": "compressed",
            "extensionName": ".min.css",
            "savePath": "/external/css",
            "removePrefix": "dev-" // Added option
        },
    ],
    "liveSassCompile.settings.includeItems": [
        "/styling/scss/dev-style.scss"
    ]
}

The above will output "style.css" as the "dev-" is removed.
We then have customizable entry point without changing core files of an application for example.

**Comment by glenn2223** [comment]
TBH I'm not too keen on the ability to tweak the output file name. This is because there could be so many hedge cases that implementing this could cause so many more to be added - causing bloat, possible performance issues and making the code even harder to maintain.

Could using `savePathSegmentKeys` and `savePathReplaceSegmentsWith` help?
- `savePathSegmentKeys` - a `string[]` of folder names
- `savePathReplaceSegmentsWith` - what to replace any "key" with

You could then have

```JSON
{
    "liveSassCompile.settings.formats":
    [
        {
            "format": "expanded",
            "extensionName": ".css",
            "savePathSegmentKeys": ["scss"],
            "savePathReplaceSegmentsWith": "dev"
        },
        {
            "format": "compressed",
            "extensionName": ".min.css",
            "savePathSegmentKeys": ["scss"],
            "savePathReplaceSegmentsWith": "external"
        },
    ],
    "liveSassCompile.settings.includeItems": [
        "/styling/scss/style.scss"
    ]
}
```

Doing this would then then output two files:
- `/styling/dev/style.css` (in an expanded format) and
- `/styling/external/style.min.css` (in a compressed format)

**Comment by RickR2H** [comment]
Thanks for your quick response! I found a bit of a hacky way which works brilliantly!

I use the SCSS extension to compile the Joomla SCSS files. I made a clone of the new Cassiopeia template in which I'm working. If there are core updates, I can just copy and paste the whole SCSS folder to get the updates. Problem is that I'm changing the template.scss entry point file. If I copy the whole folder, this file will also be overwritten.

I created a new entry point file which I called temp.scss. i modified the suffix to add the rest of the file name + extension.

`
{
    "liveSassCompile.settings.formats":[
        {
            "format": "compressed",
            "extensionName": "late.min.css",
            "savePath": "/templates/rbs5/css"
        },
    ],
    "liveSassCompile.settings.includeItems": [
        "/templates/rbs5/scss/temp.scss"
    ]
}
`
temp.scss becomes template.min.css

Thanks for the help!
